src/main/java/it/pagopa/interop/probing/scheduler/dto/PollingActiveEserviceResponse.java

```diff
@@ -0,0 +1,29 @@
+package it.pagopa.interop.probing.scheduler.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import it.pagopa.interop.probing.scheduler.util.EserviceTechnology;
+import lombok.Data;
+
+
+@Data
+public class PollingActiveEserviceResponse {
+
+  @JsonProperty("id")
+  private String id;
+
+  @JsonProperty("eserviceName")
+  private String eserviceName;
+
+  @JsonProperty("producerName")
+  private String producerName;
+
+  @JsonProperty("versionNumber")
+  private Integer versionNumber;
```

Comment:
are these information actually required for the polling?
```suggestion
```

I would expect only these fields:
- db record `id`, to update the record
- `technology`, to know how to call the service
- `basePath`, to know where to call the service

What are other fields used for?

Reply:
Ok, i will delete them. I just aligned my dto to the one of the first version of the api exposed by operations. I will inform them of the changes.

Reply:
generally, I expect each service to use only the portion of data that it requires for its business logic